### DISCUSSION TITLE
what is the difference beteen INFORMATION_SCHEMA and information_schema

### DISCUSSION THREAD

**Discussion by flyly0755** [ClickHouse/ClickHouse#33548] [Q&A] [answered]
I know both INFORMATION_SCHEMA and information_schema are something related with metadata,
but why with both and with same name but different case(one is uppercase and the other is lowercase)? the reason is?

**Comment by UnamedRus** [comment, marked as answer]
Because it's possible to have case-insensitive table and databases names in MYSQL and some  software rely on this.


https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html
